app/src/views/GrantRound.vue

```diff
@@ -44,7 +44,7 @@
         </InputRow>
 
         <!-- Submit and cancel buttons -->
-        <div class="flex justify-end pt-6">
+        <div v-if="roundStatus === ''" class="flex justify-end pt-6">
```

Comment:
Thoughts on swapping the if and else conditions.

```
<div v-if="roundStatus">...</div>
<div v-else>...</div>
```

Reply:
was matching what was done in `cart.ts` but I like your suggestion